lib/scope.js

```diff
@@ -110,15 +114,11 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
     var in_destructuring = null;
     var in_export;
     var tw = new TreeWalker(function(node, descend){
-        var create_a_block_scope =
-            (options.screw_ie8 && node instanceof AST_Catch) ||
```

Comment:
Hmm... the undefined test case is acting a bit weird in chrome. It doesn't seem that undefined can be used as variable even though it has been set as parameter.